custom/icds_reports/ucr/data_sources/dashboard/dashboard_growth_monitoring.json

```diff
@@ -0,0 +1,315 @@
+        "type": "expression",
+        "expression": {
+          "type": "nested",
+          "value_expression": {
+            "type": "property_name",
+            "property_name": "_id"
+          },
+          "argument_expression": {
+            "type": "ancestor_location",
+            "location_id": {
+              "value_expression": {
+                "type": "property_path",
+                "property_path": [
+                  "user_data",
+                  "commcare_location_id"
+                ]
+              },
+              "type": "related_doc",
+              "related_doc_type": "CommCareUser",
+              "doc_id_expression": {
+                "expression": {
+                  "type": "property_path",
+                  "property_path": [
+                    "form",
+                    "meta",
+                    "userID"
+                  ]
+                },
+                "type": "root_doc"
+              }
+            },
+            "location_type": "state"
+          }
+        },
+        "create_index": true
+      },
+      {
+        "column_id": "received_on",
+        "datatype": "datetime",
+        "type": "expression",
+        "expression": {
+          "type": "root_doc",
+          "expression": {
+            "type": "property_name",
+            "datatype": "datetime",
+            "property_name": "received_on"
+          }
+        }
+      },
+      {
+        "column_id": "child_health_case_id",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
+          "type": "conditional",
+          "test": {
+            "type": "named",
+            "name": "is_delivery_form"
+          },
+          "expression_if_true": {
+            "type": "property_path",
+            "property_path": ["case_open_child_health_3", "case", "@case_id"]
+          },
+          "expression_if_false": {
+            "type": "root_doc",
+            "expression": {
+              "type": "property_path",
+              "property_path": ["form", "case_child_health_0", "case", "@case_id"]
+            }
+          }
+        }
+      },
+      {
+        "column_id": "weight_child",
+        "datatype": "decimal",
+        "type": "expression",
+        "expression": {
+          "type": "conditional",
+          "test": {
+            "type": "named",
+            "name": "is_delivery_form"
+          },
+          "expression_if_true": {
+            "type": "property_path",
+            "property_path": ["case_open_child_health_3", "case", "update", "weight_child"]
+          },
+          "expression_if_false": {
+            "type": "root_doc",
+            "expression": {
+              "type": "property_path",
+              "property_path": ["form", "weight_child"]
+            }
+          }
+        }
+      },
+      {
+        "column_id": "height_child",
+        "datatype": "decimal",
+        "type": "expression",
+        "expression": {
```

Comment:
Delivery forms don't capture height, which is why there isn't a conditional here